source/cli.js

```diff
@@ -22,6 +22,7 @@ const cli = meow(`
 
 	Options
 	  --any-branch        Allow publishing from any branch
+	  --default-branch    Branch name to set as default branch (defaults to master)
```

Comment:
I'm not sure how I feel about the flag name (as `--branch` seems more intuitive and readable to me). @sindresorhus What do you think?

Reply:
Also, I'd rephrase this to:

```suggestion
	  --default-branch    Publish to the given branch (defaults to master)
```

Reply:
Neither of these descriptions makes it clear that *only* that branch is allowed.

Reply:
I'm fine with using `--branch`. Although, most will set this in their `np` config and not in the CLI.